dissect/target/helpers/hashutil.py

```diff
@@ -58,70 +54,34 @@ def custom(fh: BinaryIO, algos: list[Union[str, HASH]]) -> tuple[str]:
     return _hash(fh, ctx)
 
 
-def hash_uri_records(target: Target, record: Record) -> Record:
-    """Hash uri paths inside the record."""
+def hash_uri(target: Target, path: str) -> tuple[str, str]:
+    """Hash the target path."""
     warnings.warn(
         (
-            "The hash_uri_records() function is deprecated, and will be removed in dissect.target 3.15. "
-            "Use hash_path_records() instead"
+            "The hash_uri() function is deprecated, and will be removed in dissect.target 3.15."
```

Comment:
```suggestion
            "The hash_uri() function is deprecated, and will be removed in dissect.target 3.15. "
```

The space should have been there as it's a continuation.